Tests/Common/Service/UserAgentUtilTest.swift

```diff
@@ -25,7 +25,10 @@ class UserAgentUtilTest: UnitTest {
     override func setUp() {
         super.setUp()
 
-        userAgentUtil = UserAgentUtilImpl(deviceInfo: deviceInfoMock, sdkConfig: sdkConfig)
+        var sdkConfigWithWrapperConfig = sdkConfig
+        sdkConfigWithWrapperConfig._sdkWrapperConfig = sdkWrapperConfig
```

Comment:
Do we plan to keep `sdkWrapperConfig`? I think it is better to reassess the need of both `sdkWrapperConfig` and `UserAgentUtil`. If they aren't required anymore, I would prefer removing all related code instead of implementing temporary fixes to address false positives in test results. In my opinion, these are more prone to errors. Failing test signals an issue that needs to be fixed. However, once it begins producing false positives, there's high risk it might be ignored unless we're actively tracking it.

Reply:
would second rehan we dont need it anymore

Reply:
[Created ticket](https://linear.app/customerio/issue/MBL-137/remove-unneeded-sdkwrapperconfig-and-useragentutil) into backlog.